apps/api/package.json

```diff
@@ -24,6 +24,7 @@
     "@unkey/keys": "workspace:^",
     "@chronark/zod-bird": "0.3.3",
     "@hono/zod-openapi": "^0.9.6",
+    "@microlabs/otel-cf-workers": "github:mehulmathur16/otel-cf-workers",
```

Comment:
IMO this should be hosted as an npm package, especially for security reasons, just if the Unkey team decides to go with this solution.